extensions/wellinks/actions/insertMemberListEvent/insertMemberListEvent.ts

```diff
@@ -52,9 +52,17 @@ const fields = {
     id: 'eventDate',
     label: 'Event Date',
     description: 'The Date/Time of the Member List Event',
-    type: FieldType.STRING,
+    type: FieldType.DATE,
```

Comment:
@AshvinP-S is this type change an additional fix? i see it included in the PR and want to make sure there there will be expected behavior.